tsconfig.json

```diff
@@ -14,12 +14,12 @@
     "sourceMap": false,
     "declaration": false,
     "noImplicitThis": true,
-    "strict": true,
     "alwaysStrict": true,
-    "noImplicitReturns": true,
-    "noUnusedLocals": false,
     "resolveJsonModule": true,
-    "skipLibCheck": true,
+    "jsx": "preserve",
+    "exactOptionalPropertyTypes": false,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
```

Comment:
Isn't that making it less strict to disable this/

Reply:
Depends if we catch it with ESLint or not. We had `noUnusedLocals` disabled explicitly before. We never had `noUnusedParameters` specified and I am guessing it starts off as `false`.

Reply:
Looks like we don't lint for it right now. Turning it on (via inheritance).

![CleanShot 2024-12-11 at 13 42 05@2x](https://github.com/user-attachments/assets/1ccc8551-1361-40e0-a1ea-2646c87e19ff)